File: dhandho-core/src/main/java/cc/dhandho/jsclient/ng/TableJsonData.java
package cc.dhandho.jsclient.ng;

public class TableJsonData extends JsonData {

	private TableJsonData(Object json) {
		super("table",json);
	}

	public static TableJsonData valueOf(Object json) {
		TableJsonData rt = new TableJsonData(json);
		return rt;
	}

}

File: dhandho-jsclient/src/main/typings/app/common/table-json-data.ts
import { JsonData } from './json-data';

export class TableJsonData extends JsonData {
    constructor( json: any ) {
        super( 'table', json );
    }

    public static valueOf( json: any ): TableJsonData {
        const rt: TableJsonData = new TableJsonData( json );
        return rt;
    }
}
